src/seedu/addressbook/data/person/Address.java

```diff
@@ -38,19 +64,22 @@ public static boolean isValidAddress(String test) {
 
     @Override
     public String toString() {
-        return value;
+        return String.format( "%1$s,%2$s,%3$s,%4$s", this.block.toString(), this.street.toString(), this.unit.toString(), this.postalCode.toString() );
```

Comment:
Maybe you can create a new method, and then do a method call here to get the formatted string instead